packs/octopusdeploy/actions/get_deployments.py

```diff
@@ -0,0 +1,12 @@
+from lib.actions import OctopusDeployAction
+
+__all__ = [
+    'GetDeploymentsAction'
+]
+
+
+class GetDeploymentsAction(OctopusDeployAction):
+    def run(self, project_id):
+        result = self.make_get_request(action="deployments?projects="
+                                       % project_id)
```

Comment:
Would be better to expose `params` and `headers` as argument as use them instead of handcrafting the query params.
https://github.com/StackStorm/st2contrib/blob/master/packs/octopusdeploy/actions/lib/actions.py#L46